tutorials/add-disk-configure-datastore-esxi/index.mdx

```diff
@@ -1,47 +1,34 @@
 content:
   h1: How to order an additional disk and use it in ESXi as datastore
-  paragraph: This page shows how to order an additional hard disk for a Scaleway Dedibox and how to use it in ESXi as datastore.
+  paragraph: This page shows how to order an additional hard disk for a Dedibox and how to use it in ESXi as datastore.
 tags: compute server esxi data storage hard-disk disk additonal
-steps:
-  - step: Ordering an additional disk
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/add-disk-configure-fatastore-esxi/#Ordering-an-additional-disk
-  - step: Creating a virtual disk from the iDRAC interface on a hardware RAID
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/add-disk-configure-fatastore-esxi/#Creating-a-virtual-disk-from-the-iDRAC-interface-on-a-hardware-RAID
-  - step: Adding the disk as new datastore to ESXi
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/add-disk-configure-fatastore-esxi/#Adding-the-Disk-as-new-datastore-to-ESXi
-  - step: Increase the size of an existing datastore
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/add-disk-configure-fatastore-esxi/#Increase-the-size-of-an-existing-datastore
 categories:
   - dedicated-servers
   - dedibox
 image: /assets/scaleway_add_disk.webp
 dates:
-  validation: 2021-09-28
+  validation: 2022-04-04
   posted: 2020-01-21
 ---
 
 ## Additional disks with ESXi hypervisor - Overview
 
-On [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/), you can order as many [additional disks](https://www.scaleway.com/en/dedibox/storage/local-disks/) as you need. They can then, for instance, be used as additional Datastore in [ESXi](https://www.vmware.com/products/esxi-and-esx.html) to store your virtual machines.
+On [Dedibox](https://www.scaleway.com/en/dedibox/), you can order as many [additional disks](https://www.scaleway.com/en/dedibox/storage/local-disks/) as you need. They can then, for instance, be used as additional Datastore in [ESXi](https://www.vmware.com/products/esxi-and-esx.html) to store your virtual machines.
 
 A range of different SATA, SAS, and SSD disks is available. Depending on the server chassis, the number of available slots may differ.
 
 This guide explains how to order an additional disk from the Dedibox console and how to use it as datastore in a local ESXi environment by following these steps:
 
-Note that instead of adding additoonal disks to your server, you can also use remote storage based on our [RPN-SAN](https://www.scaleway.com/en/dedibox/storage/rpn-san/) product to add an additonal datastore to your ESXi host.
+Note that instead of adding additoonal disks to your server, you can also use remote storage based on our [RPN-SAN](https://www.scaleway.com/en/dedibox/storage/rpn-san/) product to add an additional datastore to your ESXi host.
```

Comment:
mettre ça dans une message box?
```suggestion
Note that instead of adding additional disks to your server, you can also use remote storage based on our [RPN-SAN](https://www.scaleway.com/en/dedibox/storage/rpn-san/) service to add an additional datastore to your ESXi host.
```